mapflow/functional/layer_utils.py

```diff
@@ -610,4 +619,4 @@ def set_raster_extent_error_handler(self,
         """
         self.add_layer(vector)
         self.iface.setActiveLayer(vector)
-        self.iface.zoomToActiveLayer()
+        self.iface.zoomToActiveLayer()
```

Comment:
Why did you remove the empty line at the end? The blank line is strongly recommended (it is default file ending in UNIX systems)